Resources/public/less/layout.less

```diff
@@ -375,26 +384,6 @@ body {
     height: 100%;
     width: 100%;
     display: block;
-    background-color: #bcbcbc;
```

Comment:
This doesn't need any background anymore?

Reply:
The empty div #headerimg is no longer used at all. It was only there to display a background.
I hadn't removed it before because it still exists in customer projects. However, it is no longer used there either, or explicitly without a background.
So I have now removed it completely in the core with a new commit.